mandatory/3-function-output.js

```diff
@@ -1,16 +1,25 @@
+/** @format */
+
 // Add comments to explain what this function does. You're meant to use Google!
+
+//This getRandomNumber() gives a random number between 0 and 10
 function getRandomNumber() {
-  return Math.random() * 10;
+	return Math.random() * 10; //Math.random will return a number between 0 and 1
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+	//combine two words with combine two strings and return one string
+	return word1.concat(word2); //The concat function concatenates two or more strings and returns the resulting string.
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+	// Write the body of this function to concatenate three words together.
+	// Look at the test case below to understand what this function is expected to return.
+
+	var firstWords = combine2Words(firstWord, " ");
+	var secondWords = combine2Words(secondWord, " ");
+	return firstWords + secondWords + thirdWord;
```

Comment:
This works, but you could do it in one line - .concat can take any number of parameters.